include/ignition/gui/qml/Chart.qml

```diff
@@ -0,0 +1,277 @@
+import QtQuick 2.0
+import QtCharts 2.2
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+
+Column {
+    id: main
+    property int chartID: -1
+    property bool multiChartsMode: false
+
+    signal subscribe(real Id, string topic, string path);
+    signal unSubscribe(real Id, string topic, string path);
+    signal clicked(real Id);
+
+    function appendPoint(_fieldID, _x, _y)
+    {
+        chart.appendPoint(_fieldID, _x, _y);
+    }
+    function setChartOpacity(_opacity)
+    {
+        chart.opacity = _opacity;
+    }
+
+    // =============== Fields info Rectangle ================
+    Rectangle{
+        id: infoRect
+        width: parent.width
+        height: parent.height / 10
+        color: (Material.theme == Material.Light) ? Material.color(Material.Grey,Material.Shade200)
+                                                  : Material.color(Material.BlueGrey, Material.Shade800)
+        // make it scrolable
+        ScrollView {
+            anchors.fill: parent
+            ScrollBar.horizontal.policy: ScrollBar.AsNeeded
+            ScrollBar.vertical.policy: ScrollBar.AlwaysOff
+            clip: true
+            // Horizontal Layout for the fields
+            Row {
+                anchors.fill: parent
+                id:row
+                spacing: 10
+            }
+        }
+
+        DropArea {
+            anchors.fill: parent
+            onDropped:
+            {
+                // topic and path is separated with ','
+                var topic_path = drop.text.split(",");
```

Comment:
We should do some checking to see if `drop.text` has a `,`. If it doesn't, print a warning and return.